g3/git/git.py

```diff
@@ -0,0 +1,21 @@
+from pydantic import BaseModel
+
+from g3.git.shell import Shell
+
+
+class GitInfo(BaseModel):
+    repo_owner: str
+    repo: str
+    branch: str
```

Comment:
Can we add a `default_branch` @property that will return `main` if found, else `master` if found else None? This could help so that we suggest to our users the target branch of our PRs.

Reply:
we were planning to add this functionality to the github client to facilitate the open pr functionality